client/e2e/poms/operations.ts

```diff
@@ -25,7 +49,106 @@ export class OperationsPOM {
 
   async urlIsCorrect() {
     const path = this.url;
-    const currentUrl = await this.page.url();
-    await expect(currentUrl.toLowerCase()).toMatch(path.toLowerCase());
+    const currentUrl = this.page.url();
+    expect(currentUrl.toLowerCase()).toMatch(path.toLowerCase());
+  }
+
+  async operationsTableIsVisible() {
+    await this.page.waitForSelector(".MuiDataGrid-root");
+  }
+
+  async operationsViewIsCorrect(role: string, expectedStatuses: string[]) {
+    let statusIndex = 5;
+    switch (role) {
+      case UserRole.INDUSTRY_USER_ADMIN:
+      case UserRole.INDUSTRY_USER:
+        // The status column is different for industry users
+        statusIndex = 4;
+        break;
+      default:
+        await expect(this.page.getByText(this.internalNote)).toBeVisible();
```

Comment:
This appears to be all new because the test file was named incorrectly and wasn't running in the playwright tests. You can see at the bottom of the changed files the deleted `industry_user_admin.ts` as I had to rename it to `industry_user_admin.spec.ts`.